ryot/src/sprites/error.rs

```diff
@@ -0,0 +1,42 @@
+/*
+ * Ryot - A free and open-source MMORPG server emulator
+ * Copyright (©) 2023 Lucas Grossi <lucas.ggrossi@gmail.com>
+ * Repository: https://github.com/lgrossi/Ryot
+ * License: https://github.com/lgrossi/Ryot/blob/main/LICENSE
+ * Contributors: https://github.com/lgrossi/Ryot/graphs/contributors
+ * Website: https://github.com/lgrossi/Ryot
+ */
```

Comment:
The auto import feature on my editor keeps adding imports above this comment. Do we need to sign off every file with this?

Reply:
I think we can drop it and stop adding them